graphene_django/types.py

```diff
@@ -109,10 +114,38 @@ def __init_subclass_with_meta__(
                 )
             )
 
+        assert not (fields and exclude), (
+            "Cannot set both 'fields' and 'exclude' options on "
+            "DjangoObjectType {class_name}.".format(class_name=cls.__name__)
+        )
+
+        # Alias only_fields -> fields
+        if only_fields and fields:
+            raise Exception("Can't set both only_fields and fields")
+        if only_fields:
+            fields = only_fields
+        if fields and fields != ALL_FIELDS and not isinstance(fields, (list, tuple)):
+            raise TypeError(
+                'The `fields` option must be a list or tuple or "__all__". '
+                "Got %s." % type(fields).__name__
```

Comment:
Minor note - we could just use f-strings now that we're only supporting Py3 on the main branch?

Reply:
@phalt I think it would be good to get this feature out before v3 and then for v3 we can go through and use f strings everywhere.

Reply:
Let's leave this for V3 then 👍